### PR TITLE
Modified Scala Getting Started example to work both locally and on MSF

### PR DESCRIPTION
**Pull request by nicusX**
Downgraded Scala version to 2.12
Added required dependencies to run locally

### CODE REVIEW

File: scala/GettingStarted/README.md [hunk 1]
```diff
@@ -25,12 +27,19 @@ Apache Flink.
...
 
-| Group ID       | Key           | Description              |
-|----------------|---------------|--------------------------|
-| `InputStream0` | `stream.name` | Name of the input stream |
-| `InputStream0` | `aws.region`  | Region of the input stream, e.g. `us-east-1` |
+| Group ID        | Key           | Description                                   |
+|-----------------|---------------|-----------------------------------------------|
+| `InputStream0`  | `stream.name` | Name of the input stream                      |
+| `InputStream0`  | `aws.region`  | Region of the input stream, e.g. `us-east-1`  |
+| `OutoutStream0` | `stream.name` | Name of the output stream                     |
```
Comment (hlteoh37) [inline scala/GettingStarted/README.md:34]: typo: Output*

File: scala/GettingStarted/build.sbt [hunk 2]
```diff
@@ -4,21 +4,28 @@ lazy val root = (project in file(".")).
...
   "com.amazonaws" % "aws-kinesisanalytics-runtime" % msfRuntimeVersion,
   "org.apache.flink" % "flink-connector-kinesis" % connectorVersion,
   "org.apache.flink" % "flink-streaming-java" % flinkVersion % "provided",
+  "org.apache.flink" %% "flink-streaming-scala" % flinkVersion % "provided",
   "org.apache.flink" % "flink-connector-base" % flinkVersion % "provided",
+  "org.apache.flink" % "flink-clients" % flinkVersion % "provided",
+  // Logging
+  "org.apache.logging.log4j" % "log4j-slf4j-impl" % log4jVersion,
+  "org.apache.logging.log4j" % "log4j-api" % log4jVersion,
+  "org.apache.logging.log4j" % "log4j-core" % log4jVersion,
```
Comment (hlteoh37) [inline scala/GettingStarted/build.sbt:28]: These should also be provided on MSF I thinks.

File: scala/GettingStarted/src/main/scala/com/amazonaws/services/msf/BasicStreamingJob.scala [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      .build
+  }
+
+  def main(args: Array[String]): Unit = {
+    val environment = StreamExecutionEnvironment.getExecutionEnvironment
+
+    val kinesisSink = createSink(environment)
+    environment
+      .addSource(createSource(environment))
+      .sinkTo(kinesisSink)
```
Comment (hlteoh37) [inline scala/GettingStarted/src/main/scala/com/amazonaws/services/msf/BasicStreamingJob.scala:55]: nit, might read better of we write

```
environment
      .addSource(createSource(environment))
      .sinkTo(createSink(environment))
```

**Comment by nicusX** [comment]
@hlteoh37 I addressed the comments

**Pull request merged**